### PR TITLE
[PROD-390] Deploy triggers build on member userpool repos

### PR DESCRIPTION
**Pull request by semiautomatixfu**
[PROD-390]

- if `inf.member.hooks` is rebuilt it will trigger a build on other listed Jenkins jobs
- at the moment, this should log an error because `inf.member.fu` has not been made available to Jenkins

[PROD-390]: https://johnbridgeman.atlassian.net/browse/PROD-390?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -58,6 +72,20 @@ node('fu && node') {
...
                 deployTo('/usr/local/bin/assume_prod.sh', stageName, 'prod')
+                try {
+                    // rebuild inf.member.* repos
+                    def jobs = ['inf.member.fu'];
+                    jobs.each { 
+                        build(
+                            job: "JB-AssetChain/${it}/${env.BRANCH_NAME}", // expect master
+                            propagate: false, 
+                            wait: false
+                        )
```
Comment (tleed5) [inline Jenkinsfile:83]: Shouldn't this have a `parameters` option for `DEPLOY_PROD` / `DEPLOY_DEV` ?

File: src/config/Config.ts [hunk 2]
```diff
@@ -1,13 +1,10 @@
 // @flow
```
Comment (tleed5) [inline src/config/Config.ts:1]: Are the`// @flow` needed anymore since it's now typescript?

**Review by tleed5 — CHANGES_REQUESTED** [review]

File: src/config/Config.ts (hunk 2 above)
Comment (semiautomatixfu) [reply src/config/Config.ts:1]: :+1:

File: Jenkinsfile (hunk 1 above)
Comment (semiautomatixfu) [reply Jenkinsfile:83]: :open_mouth: didn't even notice that they were `params` and not `env`

**Review by tleed5 — APPROVED** [review]

**Pull request merged**